.github/workflows/release-cli.yml

```diff
@@ -86,9 +91,140 @@ jobs:
           name: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
           path: bin/*
 
+  package-deb:
+    name: Build deb package
+    needs: build
+    strategy:
+      matrix:
+        include:
+          - GOOS: linux
+            GOARCH: amd64
+            pkgArch: amd64
+          - GOOS: linux
+            GOARCH: arm64
+            pkgArch: arm64
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: bin
+          pattern: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
+          merge-multiple: true
+
+      - name: create dir structure for .deb package
+        run: |
+          mkdir -p .debpkg/usr/bin
+
+          # when we download from deb, the binary should be named kubetail
+          mv 'bin/kubetail-${{ matrix.GOOS }}-${{ matrix.GOARCH }}' bin/kubetail 
+          cp -p bin/kubetail .debpkg/usr/bin
+
+      - name: create a .deb package
+        uses: jiro4989/build-deb-action@v4
+        with:
+          package: kubetail
+          package_root: .debpkg
+          maintainer: Andres Morey
+          version: ${{ steps.tagName.outputs.tag }}
+          arch: ${{ matrix.pkgArch }}
+          desc: "Real-time logging dashboard for Kubernetes"
+
+      - name: Calculate sha256 checksum
+        run: |
+          FILE='./kubetail_${{ steps.tagName.outputs.tag }}_${{ matrix.pkgArch }}.deb'
+          OUTPUT_FILE=${FILE}.sha256
+          sha256sum "$FILE" | cut -d " " -f 1 >"$OUTPUT_FILE"
+          mkdir -p deb-output
+          mv ./kubetail* ./deb-output/
```

Comment:
This line gets executed in the root of the repository so if we add any files that begin with "kubetail" in the future they will get uploaded too.

Reply:
that makes sense, and then we'll be wondering why random files are showing up on the release.